test/models/passwordless/session_test.rb

```diff
@@ -46,11 +46,46 @@ def create_session(attrs = {})
       refute_nil session.expires_at
       refute_nil session.timeout_at
       refute_nil session.token
+      refute_nil session.token_digest
+    end
+
+    test "it generates a digest" do
+      session = Session.new
+      session.validate
+
+      assert_equal 64, session.token_digest.length
+      refute_equal session.token, session.token_digest
+    end
+
+    test "with a custom digest algorithm" do
+      old_digest_algorithm = Passwordless.digest_algorithm
+
+      Passwordless.digest_algorithm = "SHA1"
+
+      session = Session.new
+      session.validate
+
+      assert_equal 40, session.token_digest.length
+
+      Passwordless.digest_algorithm = old_digest_algorithm
+    end
+
+    test "with a custom digest secret" do
```

Comment:
This test doesn't make as much sense in the session model test, but I wasn't sure where else to put it, or how to test this feature via the session model.